env.TEMPLATE

```diff
@@ -28,6 +28,10 @@ API_URL=
 AUTH_URL=
 CDN_URL=
 
+# Sentry details for logging
+SENTRY_DNS=
+SENTRY_DEBUG=
```

Comment:
What does SENTRY_DEBUG do? What are the expected values?

Reply:
SENTRY_DEBUG just logs stuff to console when creating and ending the transactions. Expects true/false